src/protagonist/API/Features/DeliveryChannels/Validation/DeliveryChannelPolicyDataValidator.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Text.Json;
+using DLCS.Core.Collections;
+using DLCS.Model.Assets;
+using IIIF.ImageApi;
+
+namespace API.Features.DeliveryChannels.Validation;
+
+public class DeliveryChannelPolicyDataValidator
+{
+    public bool Validate(string policyDataJson, string channel)
+    {
+        return channel switch
+        {
+            AssetDeliveryChannels.Thumbnails => ValidateThumbnailPolicyData(policyDataJson),
+            AssetDeliveryChannels.Timebased => ValidateTimeBasedPolicyData(policyDataJson),
+            _ => false // This is only for thumbs and iiif-av for now
+        };
+    }
+    
+    private string[]? ParseJsonPolicyData(string policyDataJson)
+    {
+        string[]? policyData;
+        try
+        {
+            policyData = JsonSerializer.Deserialize<string[]>(policyDataJson);
+        }
+        catch(JsonException ex)
```

Comment:
Don't need `ex` here as it's unused

Reply:
Would it be worth logging failed deserialisations here?